scripts/preprocessing/03-make_epochs.py

```diff
@@ -16,14 +16,25 @@
 import mne
 from mne.parallel import parallel_func
 from mne_bids import BIDSPath
-
+from autoreject import get_rejection_threshold
```

Comment:
You need to include the `autoreject` dependency in `tests/environment.yml` and also in `docs/source/getting_started/install.md`